tests/integration/targets/podman_image/tasks/main.yml

```
@@ -79,6 +86,18 @@
         state: absent
       register: rmi5
 
+    - name: Get image id of the target image
+      containers.podman.podman_info:
```

This should be `containers.podman.podman_image_info`
```suggestion
      containers.podman.podman_image_info:
```

---

Thanks. Fixed it on this commit: 75c5fa1f467abbb91fac341a0f4c218444720eba.